integration-tests/src/test/java/oracle/kubernetes/operator/ItJrfModelInImage.java

```diff
@@ -0,0 +1,243 @@
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+/**
+ * Simple JUnit test file used for testing Model in Image.
+ *
+ * <p>This test is used for creating domain using model in image.
+ */
+
+public class ItJrfModelInImage extends MiiBaseTest {
+  private static Operator operator;
+  private static String domainNS;
+  private static String testClassName;
+  private static StringBuffer namespaceList;
+  private static String rcuSchemaPrefix = "jrfrcu";
+  private static String rcuSchemaPass = "Oradoc_db1";
+  private static String walletPassword = "welcome1";
+  private static int dbPort;
+  private static String dbNamespace;
+  private static String dbUrl;
+  private static JrfDomain jrfdomain; 
+  
+  /**
+   * This method gets called only once before any of the test methods are executed. It does the
+   * initialization of the integration test properties defined in OperatorIT.properties and setting
+   * the resultRoot, pvRoot and projectRoot attributes.
+   */
+  @BeforeAll
+  public static void staticPrepare() {
+    namespaceList = new StringBuffer();
+    testClassName = new Object() {
+    }.getClass().getEnclosingClass().getSimpleName();
+    
+    LoggerHelper.getLocal().log(Level.INFO, "setting up properties, directories for: {0}", 
+        testClassName);
+    assertDoesNotThrow(() -> initialize(APP_PROPS_FILE, testClassName),
+        "Failed: initial setup");   
+  }
+
+  /**
+   * This method gets called before every test. It creates the result/pv root directories
+   * for the test. It also creates RCU schema, operator.
+   */
+  @BeforeEach
+  public void prepare() {
+
+    //createResultAndPvDirs(testClassName);
+    LoggerHelper.getLocal().log(Level.INFO, "Creating result/pv root directories");
+    assertDoesNotThrow(() -> createResultAndPvDirs(testClassName),
+        "Failed: createResultAndPvDirs");
+    
+    LoggerHelper.getLocal().log(Level.INFO, "Copying sample dir to the result dir");
+    assertDoesNotThrow(() -> TestUtils.exec(
+        "cp -rf " 
+         + BaseTest.getProjectRoot() 
+         + "/kubernetes/samples/scripts " 
+         + getResultDir(),
+         true),
+         "Failed: Copy sample dir to the result dir");
+    
+    //start DB and create RCU
+    dbNamespace = "db" + String.valueOf(getNewSuffixCount());
+    assertDoesNotThrow(() -> DbUtils.createNamespace(dbNamespace),
+        "Failed to create namespace: " + dbNamespace);
+    dbPort = 30011 + getNewSuffixCount();
+    dbUrl = "oracle-db." + dbNamespace + ".svc.cluster.local:1521/devpdb.k8s";
+    assertDoesNotThrow(() -> DbUtils.createDbRcu(getResultDir(), dbPort, dbUrl, 
```

Comment:
`createDbRcu` is poorly named and makes it non-obvious what that method does

Reply:
plus it seems to delete and restart the db for every test, which is inconsistent with what you said about reusing the schema

Reply:
Ok, I'm going to change the method name to setupRCUdatabase to to match "1) Set up and initialize an RCU database." in the section "Prerequisites for JRF domains" of mii doc 
https://github.com/oracle/weblogic-kubernetes-operator/blob/develop/docs-source/content/samples/simple/domains/model-in-image/_index.md#set-up-and-initialize-an-rcu-database

Reply:
For JRF domain RCU schema  can not be shared by 2 test runs. That is why we always cleanup RCU schema at the end of every test run in the unPrepare() phase. But as you encountered once, there could be leftover RCU and DB pods caused by aborted test. That's why we always delete both RCU and DB pods before creating new ones in the prepare() phase before every test run.
For this new mii test I'm verifying that during 1 test run(1 test class) one RCU schema can be shared between JRF domain restarts.

Reply:
Method name is changed.